system/syscalls.cpp

```diff
@@ -87,6 +87,7 @@ Page {
 			newthis->init(size-amount);
 		}
 		size = amount;
+		clear();
```

Comment:
This was removed for performance reason. It seems to me that it is not required after fixing the "isallzero" calls from musl malloc impl. Is this actually required?

Reply:
There was a test-case that failed without this. It seems that the mallocng is relying sometimes on specific bytes being zeroed. I'm currently investigating which bytes these are and how they have been allocated, to see if this zeroing can be done only in specific cases.